merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/PostgresResultsCellRepository.java

```diff
@@ -564,6 +564,7 @@ public void succeedWith(final SimulationResults results) {
         postSimulationResults(connection, datasetId, results, SimulationStateRecord.success());
         deleteSimulationExtent(connection, datasetId);
         transactionContext.commit();
+        logger.info("%s set simulation status to success".formatted(Duration.microseconds(System.nanoTime() / 1000)));
```

Comment:
Why was the logging statements added?

Reply:
We added them while trying to reproduce the issue so that we could get a better idea of timing. @dandelany asked me to leave them in the code as useful information for someone trying to understand the system's behavior